app/src/main/java/com/droidhats/campuscompass/MapsActivity.kt

```diff
@@ -35,6 +36,8 @@ import com.google.android.libraries.places.api.model.Place
 import com.google.android.libraries.places.widget.Autocomplete
 import com.google.android.libraries.places.widget.AutocompleteActivity
 import com.google.android.libraries.places.widget.model.AutocompleteActivityMode
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import kotlinx.android.synthetic.main.bottom_sheet_layout.*
```

Comment:
We need to fix this for build times. No starring imports please

Reply:
GOT IT ! Will change to specific import !